app/ordering_components.py

```diff
@@ -1,37 +1,33 @@
-async def get_approval_status(result, message):
-    for nbindings in result["node_bindings"].items():
-        for node in nbindings:
-            for attribute in message["knowledge_graph"]["nodes"][node["id"]]["attributes"]:
-                if attribute["attribute_type_id"] is "biothings_annotations":
-                    # This returns 1 of 5 values: 0, .25, .5, .75, 1, depending on the max phase.
-                    # TODO Determine proper appraisal of approval status
-                    max_phase = attribute["value"].get("chembl", 0).get("max_phase", 0)
-                    return max_phase/4
-            
-async def get_confidence(result, message):
+"""Compute scores for each result in the given message."""
+
+
+def get_confidence(result, message, logger):
     # TODO Implement actual g()-score
     score_sum = 0
     score_count = 0
-    for analysis in result:
+    for analysis in result["analyses"]:
```

Comment:
use get() for message parsing saftey?

Reply:
Sure. Analyses are required inside each result, but I agree it's good form to be sure.